### DISCUSSION TITLE
Credit Devin as well?

### DISCUSSION THREAD

**Discussion by punishedbrains** [Sakati84/3DPrintColorizer#57] [General]
Not sure if it was seen on the channel "Make Anything", but if so, it might be nice to credit him as well?

**Comment by Sakati84** [comment]
I´m a big fan of his channel but I don´t really see why I would do that. The way the sharpies are used is pretty different from what he did and he is by no means the inventor of this sharpie-filamant idea. Something he himself explicitly said in his video about this. There are several YT videos doing the exact same thing (ok.. only with 3 sharpies) years before his video.

**Comment by punishedbrains** [comment]
That's fine, it's just that he did show this recently and not years ago. But I get your argument as well. Thanks for your reply.

**Reply by Sakati84** [reply]
Veritasium did a video about gravity recently. I would still credit Newton though..
